classes/data-stores/ActionScheduler_HybridStore.php

```diff
@@ -324,13 +321,45 @@ public function mark_complete( $action_id ) {
 	 * @param int $action_id Action ID.
 	 */
 	public function get_status( $action_id ) {
+		$store = $this->get_store_from_action_id( $action_id );
+		if ( $store ) {
+			return $store->get_status( $action_id );
+		}
+		return null;
+	}
+
+	/**
+	 * Return which store an action is stored in.
+	 *
+	 * @param int  $action_id ID of the action.
+	 * @param bool $primary_first Optional flag indicating search the primary store first.
+	 * @return ActionScheduler_Store
+	 */
+	protected function get_store_from_action_id( $action_id, $primary_first = false ) {
 		if ( $action_id < $this->demarkation_id ) {
-			return $this->secondary_store->get_status( $action_id );
+			$stores = [
+				$this->primary_store,
+				$this->secondary_store,
+			];
```

Comment:
Should this order be reversed to match the original logic - or is the order change intentional?

Reply:
This is intentional.